### DISCUSSION TITLE
Hide the content in my Verdaccio registry page

### DISCUSSION THREAD

**Discussion by WangJiaoJo** [verdaccio/verdaccio#3952] [Q&A]
Hi, team,
I can disable the download functionality by configuring the property of ‘showDownloadTarball’ in the config.yaml file.
Is there any way that I can hide all the content in my Verdaccio registry page? Then I still can fetch the NPM packages but just now be able to see them on the webpage.
Thanks in advance!

**Comment by juanpicado** [comment]
> Is there any way that I can hide all the content in my Verdaccio registry page? 

Do you mean hide the whole website? https://verdaccio.org/docs/webui

If that's the question you can use

```
web:
  enable: false
```
